python/pipeline/meso.py

```diff
@@ -1040,7 +1040,7 @@ def update_img(i):
             print('If this takes too long, stop it and call again with dpi <', dpi, '(default)')
             video.save(filename, dpi=dpi)
 
-            return fig
+            return fig,video
```

Comment:
either document that it willl also return video and add the return for reso too to keep it consistent or only return figure. I think video dies when you leave the function and that's why I didn't return it

Reply:
video removed from return